Docs/seo updates (#36083)
hashicorp/terraform: website/docs/cli/commands/state/index.mdx

---
page_title: terraform state commands reference

> description: The `terraform state` group of commands enable advanced Terraform state management.

Reviewer: ```suggestion
description: The `terraform state` command and subcommands enable advanced Terraform state management.
```
